### DISCUSSION TITLE
There are many AI projects and frameworks on the market. Why should KT be packaged separately?

### DISCUSSION THREAD

**Discussion by Richyeoh** [Kotlin/kotlindl#295] [General]
I just be curious!

**Comment by zaleslaw** [comment]
This project just starts to give access to the Keras-like API for the JVM developers based on the TensorFlow runtime with the ability to make transfer-learning and training from scratch for the large models. After that, we decided to help the teams who want to avoid a painful porting of numpy/torch/tensorflow/onnx pieces of code to the JVM ecosystem

No such framework on the JVM/Kotlin market could decide this problem fully in the near future.
